src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
```

Comment:
Missing a description or purpose of the class in the Javadoc comment. (Severity: low)